codegen/templates/templates.go

```diff
@@ -206,6 +206,7 @@ func Funcs() template.FuncMap {
 		"call":               Call,
 		"prefixLines":        prefixLines,
 		"notNil":             notNil,
+		"strSplit":           StrSplit,
```

Comment:
Do we need to cover this with a test?

Reply:
It's actually calling go standard library's `strings.Split` function only.

I would ask some test scenarios If maintainers ask it.

Reply:
According to a [PR](https://github.com/99designs/gqlgen/blob/master/.github/PULL_REQUEST_TEMPLATE.md) requirement, every new feature should have a test.

It doesn't matter how simple the function's implementation is now, someone may change its implementation during a refactoring, and we won't catch this without a test.

See